### PR TITLE
FIx spelling of the Ukrainian language

### PR DESCRIPTION
**Pull request by gmikhail**
Now with a capital letter, like other languages

### CODE REVIEW

**Comment by peppy** [comment]
Is there a reason it doesn't have a capital in the [wikipedia listing](https://en.wikipedia.org/wiki/Ukrainian_language)?

**Comment by tybug** [comment]
For what it's worth, other wikipedia pages don't have the native language name capitalized either. Eg https://en.wikipedia.org/wiki/Spanish_language and https://en.wikipedia.org/wiki/French_language. I'm not sure why.
image

**Comment by gmikhail** [comment]
> Is there a reason it doesn't have a capital in the [wikipedia listing](https://en.wikipedia.org/wiki/Ukrainian_language)?

Because articles in foreign languages are often inaccurate. 

If you mean the information block on the right - Russian language is also written with a small letter in [English wikipedia](https://en.wikipedia.org/wiki/Russian_language). But in the native language page the language title is capitalized:
[https://uk.wikipedia.org/wiki/Українська_мова](https://uk.wikipedia.org/wiki/Українська_мова)
[https://ru.wikipedia.org/wiki/Русский_язык](https://ru.wikipedia.org/wiki/Русский_язык)

TL;DR - I'm native Ukrainian and Russian speaker. Trust me 😉

In addition, all languages on the wesite use capital letters, except for Ukrainian, so this is just a unification.

**Comment by lmnyx** [comment]
Wikipedia writes the romanization on non-native version of the website of the *cyrillic* language name lowercase for some reason.
Examples:
https://en.wikipedia.org/wiki/Belarusian_language
https://en.wikipedia.org/wiki/Russian_language
https://en.wikipedia.org/wiki/Ukrainian_language
https://en.wikipedia.org/wiki/Bulgarian_language
